ICSharpCode.Decompiler/FlowAnalysis/SsaVariable.cs

```diff
@@ -44,7 +44,7 @@ public SsaVariable(ParameterDefinition p)
 		
 		public SsaVariable(VariableDefinition v)
 		{
-			this.Name = string.IsNullOrEmpty(v.Name) ? "V_" + v.Index : v.Name;
+			this.Name = v.ToString();
```

Comment:
Not honestly sure where this Ssa stuff comes into play in the app (I'd just dabbled when I made my branch), but along the same lines as everything else, I added a method parameter for this and passed in the value [over here](https://github.com/airbreather/ILSpy/commit/4d18cd33e29de480f172b0b23aa4a1fefab845fa#diff-03e40afd8d099655eb718527eca113cfR87).